lib/_stream_duplex.js

```diff
@@ -61,6 +61,12 @@ function Duplex(options) {
   this.once('end', onend);
 }
 
+Object.defineProperty(Duplex.prototype, 'writeHWM', {
```

Comment:
not really a fan of the abbreviation on this. Would prefer spelling it out e.g. `writeHighWaterMark`